tka/state.go

```diff
@@ -249,6 +249,8 @@ func (s *State) staticValidateCheckpoint() error {
 		if err := k.StaticValidate(); err != nil {
 			return fmt.Errorf("key[%d]: %v", i, err)
 		}
+	}
+	for i, k := range s.Keys {
```

Comment:
maybe add a comment saying this quadratic loop is fine because s.Keys is required to be small (<1000)

Reply:
Done.